x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/__snapshots__/list.test.tsx.snap

```diff
@@ -61,33 +61,6 @@ exports[`CustomUrlList renders a list of custom URLs 1`] = `
         />
       </EuiFormRow>
     </EuiFlexItem>
-    <EuiFlexItem
```

Comment:
I'd expect these snapshots not to change what do you think? Maybe it's worth updating the configs in the tests so they render the same.

Reply:
Yep - it wasn't recognizing the job as an anomaly detection job so was removing the time range selector. I just added in the properties so it would be recognized and updated the result 👍  in 7ee455c02c2252a159431305bf8b90cabaed6305